iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/MWMTaxiPreviewCell.h

```diff
@@ -1,10 +1,10 @@
-namespace uber
+namespace taxi
 {
-  struct Product;
-}  // namespace uber;
+struct Product;
+}  // namespace taxi;
```

Comment:
The usual notation is to avoid trailing comments like this in headers, when namespaces are written for forward-declarations only.